app/models/atmosphere/action.rb

```diff
@@ -0,0 +1,22 @@
+module Atmosphere
+  class Action < ActiveRecord::Base
+    belongs_to :appliance,
+               class_name: 'Atmosphere::Appliance'
+
+    has_many :action_logs, -> { order(id: :asc) },
+             dependent: :destroy,
```

Comment:
We were discussing about it with @bwilk and decided that right now yes. In the feature we can use e.g. [paranoia](https://github.com/radar/paranoia) for soft appliance and actions destroy. Appliances older than e.g. month will be removed permanently.